src/roadmap/v1/lifecycle/app_streams.py

```diff
@@ -372,6 +372,9 @@ def app_streams_from_modules(
         if os_major not in get_module_os_major_versions(module_name):
             continue
 
+        if os_major <= 8 and "installed" not in dnf_module.get("status", []):
+            continue
+
```

Comment:
This isn't a complete solution since RHEL 9 or later systems that have modules that are not installed will incorrectly show up in the results. The expression after `and` isn't evaluated because `os_major <= 8` is `False`.